pvlib/iotools/acis.py

```diff
@@ -0,0 +1,483 @@
+import requests
+import pandas as pd
+import numpy as np
+
+
+VARIABLE_MAP = {
+    # time series names
+    'pcpn': 'precipitation',
+    'maxt': 'temp_air_max',
+    'avgt': 'temp_air_average',
```

Comment:
Perhaps remove `_average` if this is almost always the temperature of interest for PV applications? For example, the bsrn function doesn't append `_average` but does append `_min` and `_max`.

Reply:
I lean towards keeping the `_average` suffix here.  Daily values seem different enough from the higher-resolution values that we call `temp_air` that consistency with the other names in this dict seems more important to me.  It doesn't make sense to feed these values into `ModelChain.run_model`, for example.